src/zone.ts

```diff
@@ -358,6 +375,17 @@ class Zone extends ServiceObject<Zone> {
     this.getChangesStream = paginator.streamify('getChanges');
   }
 
+  create(config: CreateZoneRequest): Promise<CreateZoneResponse>;
+  create(config: CreateZoneRequest, callback: CreateZoneCallback): void;
+  create(
+    config: CreateZoneRequest,
+    callback?: CreateZoneCallback
+  ): void | Promise<CreateZoneResponse> {
+    // tslint:disable-next-line no-any
+    const args = [config, callback!] as any;
+    ServiceObject.prototype.create.apply(this, args);
```

Comment:
Any reason why this couldn't be:

```js
return super.create(config, callback);
```

Reply:
My approach here was to make TypeScript think there wasn't a `create` method in the prototype chain so that we could override it with the correct signature. As a result we can't call `super` here because it thinks that method doesn't exist.